service/src/test/java/bio/terra/lz/futureservice/pact/consumer/SamServiceConsumerPactTest.java

```diff
@@ -0,0 +1,187 @@
+                "actions": [
+                  "string"
+                ]
+              },
+              "inherited": {
+                "roles": [
+                  "string"
+                ],
+                "actions": [
+                  "string"
+                ]
+              },
+              "public": {
+                "roles": [
+                  "string"
+                ],
+                "actions": [
+                  "string"
+                ]
+              },
+              "authDomainGroups": [
+                "string"
+              ],
+              "missingAuthDomainGroups": [
+                "string"
+              ]
+            }
+          ]
+          """
+          .formatted(RESOURCE_ID);
+
+  //  private static final UUID BILLING_PROFILE_ID =
+  // UUID.fromString("04063380-b0f1-11ee-acde-00155df9466d");
+  //  private static final UUID LANDING_ZONE_ID =
+  // UUID.fromString("0979c4e4-b0f1-11ee-a0d3-00155df9466d");
+
+  @Pact(consumer = "lzs", provider = "sam")
+  public RequestResponsePact statusApiPact(PactDslWithProvider builder) {
+    return builder
+        .given("Sam is ok")
+        .uponReceiving("a status request")
+        .path("/status")
+        .method("GET")
+        .willRespondWith()
+        .status(200)
+        .body(new PactDslJsonBody().booleanValue("ok", true))
+        .toPact();
+  }
+
+  @Pact(consumer = "lzs", provider = "sam")
+  public RequestResponsePact resourceExistingPermissionV2Pact(PactDslWithProvider builder) {
+    return builder
+        .given("permission exists")
+        .uponReceiving("a request to check permissions")
+        .path("/api/resources/v2/%s/%s/action/%s".formatted(RESOURCE_TYPE, RESOURCE_ID, ACTION))
+        .method("GET")
+        .willRespondWith()
+        .status(200)
+        .body("true")
+        .toPact();
+  }
+
+  @Pact(consumer = "lzs", provider = "sam")
+  public RequestResponsePact userStatusInfoPact(PactDslWithProvider builder) {
+    var responseBody =
+        new PactDslJsonBody()
+            .stringType("userSubjectId")
+            .stringType("userEmail")
+            .booleanType("enabled");
+    return builder
+        .given("user status info request with access token")
+        .uponReceiving("a request for the user's status")
+        .path("/register/user/v2/self/info")
+        .method("GET")
+        .headers("Authorization", "Bearer accessToken")
+        .willRespondWith()
+        .status(200)
+        .body(responseBody)
+        .toPact();
+  }
+
+  @Pact(consumer = "lzs", provider = "sam")
+  public RequestResponsePact listLandingZoneResourceIdsPact(PactDslWithProvider builder) {
+    return builder
+        .given("landing zone resource identifiers exist")
+        .uponReceiving("a request to list landing zone resource ids")
+        .path("/api/resources/v2/%s".formatted(SamConstants.SamResourceType.LANDING_ZONE))
+        .method("GET")
+        .headers("Authorization", "Bearer accessToken")
```

Comment:
Most of the time, the `Authorization` header should just be left out of the pact, and it's just considered to be outside the scope of the testing.

For tests that rely on the specific value of the auth header, such as the user status endpoint, it's usually best to use a provider-side value associated with a specific state in the provider tests.